### DISCUSSION TITLE
Operadores Logicos

### DISCUSSION THREAD

**Discussion by cmsouzaac** [balta-io/2801#101] [Perguntas e Respostas] [answered]
```
namespace CursoCSharp.Fundamentos
{

    class logicoss
    {
        public static void Executar()
        {
            int x = 12;
            bool entre = (x > 25) && (x < 40);

            Console.WriteLine(entre);
        }

    }

}
```
@andrebaltieri estou com duvidas referente a qual condição ele retornou "false", compilador verifica uma condição ou as duas pra retornar "false". Pois 12 nao é maior que 25 / mais 12 é menor que 40.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @cmsoouza, como vai?

Neste caso ele executa ambas, é uma condição **e**, sendo assim tem que atender ambas para ser verdadeiro.
Porém, ele retorna se ambas são verdadeiras... se uma for falsa, o retorno será `false`.

No caso, 12 não é maior que 25, logo `entre` terá o valor `false`

**Reply by cmsouzaac** [reply]
agora entendi professor!
